python/UniverseWrapper.cpp

```diff
@@ -176,6 +204,20 @@ namespace {
     }
 }
 
+void ClearPathfinderCaches() {
+    DebugLogger() << "Clearing pathfinder cache. Previous hits/misses: " << hits << " / " << misses
+                  << " : " << std::to_string(static_cast<float>(hits) / (hits + misses) * 100.0f) << "%";
+    hits = 0;
+    misses = 1;
```

Comment:
Why misses are 1?

Reply:
so I don't get a divide by 0 error on the first clear and log